cypress/support/fragments/invoices/invoices.js

```diff
@@ -248,7 +248,7 @@ export default {
       PaneHeader({ id: 'paneHeaderinvoice-results-pane' })
         .find(actionsButton).click(),
       Button('Voucher export').click(),
-      Select().choose('FOLIO'),
+      Select().choose('Amherst (AC)'),
```

Comment:
may be it's good point to prepare new piece of test data?

Reply:
I'll be thinking about that?But now it it's that,what working

Reply:
what's the reason of lose of 'FOLIO'? It can work now, but may be broken after 1 minute as with FOLIO. Need to determine why Folio was disappeared and Amherst (AC) was added

Reply:
The default is batched group Amherst (AC) when you create an invoice. after that you need to choose a group for export, and I have it FOLIO. because of this I changed the choice.